stock_picking_line_sequence/models/stock.py

```diff
@@ -35,22 +35,25 @@ class StockMoveLine(models.Model):
     _inherit = "stock.move.line"
 
     def _get_aggregated_product_quantities(self, **kwargs):
+        def get_aggregated_properties(move_line=False, move=False):
+            move = move or move_line.move_id
+            uom = move.product_uom or move_line.product_uom_id
+            name = move.product_id.display_name
+            description = move.description_picking
+            if description == name or description == move.product_id.name:
+                description = False
+            product = move.product_id
+            line_key = (
+                f"{product.id}_{product.display_name}_" f'{description or ""}_{uom.id}'
+            )
+            return (line_key, name, description, uom)
+
         aggregated_move_lines = super(
             StockMoveLine, self
         )._get_aggregated_product_quantities(**kwargs)
         for move_line in self:
-            name = move_line.product_id.display_name
-            description = move_line.move_id.description_picking
-            if description == name or description == move_line.product_id.name:
-                description = False
-            uom = move_line.product_uom_id
-            line_key = (
-                str(move_line.product_id.id)
-                + "_"
-                + name
-                + (description or "")
-                + "uom "
-                + str(uom.id)
+            line_key, name, description, uom = get_aggregated_properties(
```

Comment:
Am I wrong or some variables are not used ?

Reply:
Totally true, I just remove the unused variables. Thanks!